package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "jest-junit",
-  "version": "12.3.0",
+  "version": "12.3.1",
```

```suggestion
  "version": "12.3.0",
```

@palmerj3 will bump this when making the release